### PR TITLE
Allow user defined query strings in resource

### PR DESCRIPTION
**Pull request by alexbourret**

### CODE REVIEW

File: python-lib/odata_plugin_common.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+try:
+    from urllib.parse import urlparse, parse_qs, urlunsplit
+except:
+    from urlparse import urlparse, parse_qs, urlunsplit
```
Comment (JaneBellaiche) [inline python-lib/odata_plugin_common.py]: Maybe put a comment to precise which one is for which python version for maintainability

File: python-lib/odata_client.py [hunk 2]
```diff
@@ -108,6 +113,7 @@ def extract_next_page_url(self, data):
             return None
         if self.is_next_page_an_url:
             logging.info("Next page url={}".format(next_page_token))
+            self.initial_params = None  # The initial query strings are relayed in the next page url
```
Comment (JaneBellaiche) [inline python-lib/odata_client.py:116]: Could we add a test scenario to check this case?

Comment (alexbourret) [reply python-lib/odata_client.py:116]: This case is tested by building the *V4 datasets in the `ODATA_QUERYSTRING` scenario. (and V3 datasets are testing the other case)

File: python-lib/odata_plugin_common.py (hunk 1 above)
Comment (alexbourret) [reply python-lib/odata_plugin_common.py]: 👍 Done in [19efbaf](https://github.com/dataiku/dss-plugin-odata/pull/5/commits/19efbaf6995ffc773bf4f47bb6cb81679205ee1e)

**Review by JaneBellaiche — APPROVED** [review]

**Pull request merged**